TLM/TLM/UI/Helpers/CheckboxOption.cs

```diff
@@ -24,22 +25,40 @@ public class CheckboxOption : SerializableUIOptionBase<bool, UICheckBox> {
             }
         }
 
-        public override void Load(byte data) => Value = (data != 0);
+        public override bool Enabled {
+            get => base.Enabled;
+            set {
+                base.Enabled = value;
+                if (_ui != null) {
+                    _ui.readOnly = !value;
+                    _ui.opacity = value ? 1f : 0.3f;
+                }
+            }
+        }
+
+        public override void Load(byte data) => Value = data != 0;
         public override byte Save() => Value ? (byte)1 : (byte)0;
 
         public override void AddUI(UIHelperBase container) {
-            string T(string key) => Translation.Options.Get(key);
+            bool unalterable = !GlobalOption && !Options.IsGameLoaded(false);
```

Comment:
negation in variable name and double negation in its value. Can you flip its name to "editable" or "alterable" and change the formulas to opposite?

Reply:
Yup, will do.

Reply:
see if that's any better

Reply:
This works